tools/pipelines/build-docs.yml

```diff
@@ -88,39 +88,54 @@ variables:
   - name: pnpmStorePath
     value: $(Pipeline.Workspace)/.pnpm-store
 
+# TODO: include release/* branches as trigger. Currently, the website docs builds and deploys on each commit.
+# However, inconsistencies between the release branches and main is causing the website to change unexpectedly
+# we can re include the release/* branch to the trigger after separating the deploy step to another pipeline
 # no PR triggers
 trigger:
   branches:
     include:
-    - release/*
+    # - release/*
     - main
 pr: none
 
+# TODO: the download artifact step is currently targetting the latest pipeline run from each resource.
+# However, this is causing inconsistencies in the doc in scenarios where the latest runs on each pipeline
+# is not the same (e.g. mix of main and lts). Temporarily setting the branch to main explicitly.
 resources:
   pipelines:
   - pipeline: common_definitions
     source: Build - common-definitions
+    branch: main
   - pipeline: common_utils
     source: Build - common-utils
+    branch: main
   - pipeline: container_definitions
     source: Build - container-definitions
+    branch: main
   - pipeline: core_interfaces
     source: Build - core-interfaces
+    branch: main
   - pipeline: driver_definitions
     source: Build - driver-definitions
+    branch: main
   - pipeline: protocol_definitions
     source: Build - protocol-definitions
+    branch: main
   - pipeline: azure
     source: Build - azure
+    branch: main
   - pipeline: client
     source: Build - client packages
+    branch: main
     trigger:
```

Comment:
Probably not for this PR, but I wonder why this trigger config includes branches starting with `docs/`, and why we have special-cased the client build for `docs/` branches. @tylerbutler do you happen to know? Should this be removed?